### PR TITLE
Celery crontab

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Add cronjobs using these recommendations https://github.com/dokku/dokku/blob/master/docs/deployment/one-off-processes.md#general-cron-recommendations

Using celery and rabbitmq.
Using celery beat for cronjobs and flower s admin panel that can be accessed at port 5555

- [x] Merge #31

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: contrib/start.sh [hunk 1]
```diff
@@ -1,12 +1,18 @@
 #!/bin/sh
+python manage.py makemigrations                   # Make sure feinstaub migrations are created
```
[the remark this answers is not in the document]
Comment (karimkawambwa) [reply contrib/start.sh]: Feinstaub made changes without a migration on their repo.
I guess I should push a migration to feinstaub? @kilemensi

File: docker-compose.yml [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+  rabbitmq:
+    image: bitnami/rabbitmq:latest
+    ports:
+      - 4369:4369
+      - 5672:5672
+      - 25672:25672
+      - 15672:15672
+    environment:
+      - RABBITMQ_USERNAME=sensorsafrica
+      - RABBITMQ_PASSWORD=sensorsafrica
```
[the remark this answers is not in the document]
Comment (karimkawambwa) [reply docker-compose.yml]: This is for rabbitmq image.

File: Dockerfile [hunk 3]
```diff
@@ -24,7 +24,9 @@ RUN pip install -q -U .
 
 # Expose port server
 EXPOSE 8000
+EXPOSE 5555
 
+COPY ./requirements.txt /requirements.txt
```
[the remark this answers is not in the document]
Comment (karimkawambwa) [reply Dockerfile]: This was an attempt to get requirements cache. I shouldv removed this.

File: contrib/start.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (kilemensi) [reply contrib/start.sh]: Yep @karimkawambwa. That's how we roll :sunglasses:

File: docker-compose.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (kilemensi) [reply docker-compose.yml]: The emphasis is on _our_ @karimkawambwa... For example we have `RABBITMQ_URL` in `settings.py`. This should be `SENSORSAFRICA_RABBITMQ_URL` to match with `SENSORSAFRICA_DATABASE_URL`, etc.

I agree, anything that's not in _our_ control, we should leave as is.

**Comment by karimkawambwa** [comment]
Wait for this https://github.com/opendata-stuttgart/feinstaub-api/pull/23 to get merged and rerun the tests.

**Comment by karimkawambwa** [comment]
Will move feinstaub migrations to sensorsafrica instead  @kilemensi 👀 on PR

**Comment by kilemensi** [comment]
> Will move feinstaub migrations to sensorsafrica instead @kilemensi eyes on PR

Wait, do we need to move **all** @karimkawambwa? I thought there was just one or two that were missing and hence these will be the only ones we need to maintain.

**Comment by karimkawambwa** [comment]
> Wait, do we need to move all @karimkawambwa? I thought there was just one or two that were missing and hence these will be the only ones we need to maintain.

Its best to move all migrations over because it will fail to find previous migrations. unless we squash them. @kilemensi

**Comment by kilemensi** [comment]
> > Wait, do we need to move all @karimkawambwa? I thought there was just one or two that were missing and hence these will be the only ones we need to maintain.
> 
> Its best to move all migrations over because it will fail to find previous migrations. unless we squash them. @kilemensi

Then lets rethink this approach a bit @karimkawambwa.

**Comment by karimkawambwa** [comment]
@kilemensi or @phillipahereza  👀

**Pull request merged**